src/core/guards/__tests__/base-rate-limit.guard.spec.ts

```
@@ -0,0 +1,407 @@
+          statusCode: 429,
+        },
+      });
+
+      const result = await successGuard.canActivate(mockContext);
+      expect(result).toBe(true);
+    });
+
+    it('should handle actual limiter error through mock', async () => {
+      // Create a more realistic test by mocking the actual limiter behavior
+      process.env.NODE_ENV = 'production';
+      
```

ESLint

[error] Delete `······`

(prettier/prettier)